### ISSUE TITLE
[Accessibility-Links C6.2] The notification setting link must have an accessible name

### ISSUE THREAD

**Issue by mbensalem** [Meeds-io/meeds#3366]
**Step of reproduction:**

- Access to the notification button on the top bar
-  Access to the notification settings link

**Current behavior:**

The notification settings link doesn't have an accessible name

**Expected behavior:**

Add **aria-label= "Notification settings"** to the notification settings link
